File: Python/Pattern/02.hollowrectangle.py
'''Create a program to print this

*****

*     *

*****

'''

row = int(input("Enter No. of Rows:"))
col = int(input("Enter No. of Cols:"))

for i in range (row):
    if (i == 0 or i == row-1):
        for i in range (col):
            print("*", end="")
    else:
        print("*", end= "")
        for i in range (col-2):
            print(" ", end="")
        print("*", end="")
    print("\n")
    

File: CPP/Patterns/02. hollowrectangle.cpp
/* Create a program to print this

*****

*   *

*****
*/


#include <iostream>
using namespace std;

int main( ){
    int rowCount, colCount;

    cout<<"Enter No. of rows: ";
    cin>> rowCount;

    cout<< "Enter No. of cols: ";
    cin>> colCount;

    for (int row = 0; row < rowCount ; row++){
        if (row ==0 || row == rowCount - 1){
            for (int  i = 0; i < colCount; i++){
                cout<<"* ";
            }
        }
        else{
            cout<<"* ";
            for (int  i = 0; i < colCount-2; i++)
            {
                cout<<"  ";
            }
            cout<<"* ";
        }
        cout<< "\n";
    }

    return 0;
}